cms/static/js/views/course_video_settings.js

```diff
@@ -716,9 +725,17 @@ function($, Backbone, _, gettext, moment, ViewUtils, HtmlUtils, StringUtils, Tra
                     )
                 );
             }).fail(function(jqXHR) {
-                if (jqXHR.responseText) {
-                    self.updateFailResponseStatus(jqXHR.responseText);
+                var errorMessage;
+                // Only set error message in case 400 status.
+                if (jqXHR.responseText && jqXHR.status === 400) {
+                    errorMessage = jqXHR.responseText;
+                } else {
+                    // TODO: check with @sylvia
+                    // This error happens when invalid server configuration for JWT authentication and communication
+                    // b/w platform and video-pipleine.
+                    errorMessage = gettext('Server encoutered error.');
```

Comment:
@srpearce Can you suggest this error message text?

Reply:
What can users do in this case? I'm guessing here, so let me know if this text works:

An error has occurred. Wait a few minutes, and then try again.